Source/WebCore/platform/graphics/coretext/FontCustomPlatformDataCoreText.cpp

```diff
@@ -74,19 +74,33 @@ RefPtr<FontCustomPlatformData> createFontCustomPlatformData(SharedBuffer& buffer
     if (!itemInCollection.isNull()) {
         if (auto desiredName = itemInCollection.createCFString()) {
             for (CFIndex i = 0; i < length; ++i) {
-                auto candidate = static_cast<CTFontDescriptorRef>(CFArrayGetValueAtIndex(array.get(), i));
-                auto postScriptName = adoptCF(static_cast<CFStringRef>(CTFontDescriptorCopyAttribute(candidate, kCTFontNameAttribute)));
+                auto candidate = static_cast<FPFontRef>(CFArrayGetValueAtIndex(array.get(), i));
+                auto postScriptName = adoptCF(static_cast<CFStringRef>(FPFontCopyPostScriptName(candidate)));
```

Comment:
Why is the `static_cast<CFStringRef>()` needed here when `FPFontCopyPostScriptName()` is declared to return `CFStringRef`?